infrastructure/quick-deploy/aws/all/variables.tf

```diff
@@ -514,11 +528,11 @@ variable "admin_gui" {
         memory = optional(string)
       }))
     }), {})
-    app = optional(object({
-      name  = optional(string, "admin-app")
-      image = optional(string, "dockerhubaneo/armonik_admin_app")
-      tag   = optional(string)
-      port  = optional(number, 1080)
+    old = optional(object({
+      name  = "admin-old-gui"
+      image = "dockerhubaneo/armonik_admin_app"
+      tag   = "0.8.0"
+      port  = 1080
```

Comment:
This will not work. You should just keep the optionals, and add the tag as the 2nd argument of the optional tag.